packages/commonwealth/server/routes/createChain.ts

```diff
@@ -246,6 +275,17 @@ const createChain = async (
       return next(new AppError(Errors.InvalidNode));
     }
 
+    if (cosmos_chain_id) {
```

Comment:
Can we move this error up? So we avoid making the external web query if the chain id already exists in the db.

Reply:
good call. done